SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/Base.h

```diff
@@ -527,10 +527,10 @@ class SOFA_CORE_API Base
 
     Data< sofa::core::objectmodel::ComponentState >  d_componentState; ///< the object state
 
-    SOFA_DEPRECATE_COMPONENTSTATE("To fix your code, use d_componentState")
+    SOFA_ATTRIBUTE_DISABLED("2020-04-30 PR#1358", "v20.12", "To fix your code, use d_componentState")
```

Comment:
There should be a macro SOFA_ATTRIBUTE_DISABLED_OLDCOMPONENTSTATE_NAMES instead of using a generic macro.